paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerChunkUnloadScriptEvent.java

```diff
@@ -0,0 +1,67 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.ChunkTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import io.papermc.paper.event.packet.PlayerChunkUnloadEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class PlayerChunkUnloadScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player unloads chunk
+    //
+    // @Group Paper
+    //
+    // @Location true
+    //
+    // @Plugin Paper
+    //
+    // @Warning This event will fire *extremely* rapidly and almost guarantees lag. Use with maximum caution.
+    //
+    // @Triggers when a player unloads a chunk.
+    //
+    // @Context
+    // <context.player> returns a PlayerTag of the player that unloads the chunk.
```

Comment:
Nitpick, but `player unloading the chunk`/`that unloaded` would probably be a better fit?